_data/handles/Secureverse.json

```diff
@@ -16,4 +12,5 @@
     }
   ],
   "image": "./avatars/Secureverse.png"
```

Comment:
needs to be a comma between this line and the next this otherwise it can't be parsed as JSON